cache-config/testing/docker/ort_test/run.sh

```diff
@@ -99,9 +99,9 @@ echo "waiting $WAIT seconds for all containers to initialize."
 sleep $WAIT
 
 cp /ort-tests/tc-fixtures.json /tc-fixtures.json
-(touch test.log && tail -f test.log)&
+(touch test.log && chmod a+rw test.log && tail -f test.log)&
 
-go test -v -failfast -cfg=conf/docker-edge-cache.conf 2>&1 >> test.log
```

Comment:
We intentionally added `-failfast` because it makes debugging much easier. I can see why you might want all logs in Github. But when running the tests manually, errors after the first are very frequently spurious and misleading. It's much more straightforward to see and fix the first error, and then fix the next if it still fails.